### ISSUE TITLE
Authors Since Publication

### ISSUE THREAD

**Issue by cgreene** [greenelab/deep-review#959]
I'm creating this issue to capture authors who have contributed since the published version akin to how #561 was used to capture authors who contributed during revision.

- [ ] @jlevy44

**Comment by agitter** [comment]
Good idea.  Should we also add a footnote before the affiliations in the manuscript noting additional authors with a link here?

**Comment by evancofer** [comment]
Good plan.

**Comment by jlevy44** [comment]
We should also add @brockclarke for #956

**Comment by cgreene** [comment]
@jlevy44 I was adding people as I merged the PRs. I thought the one from @alexandertitus was ready to go, until I noticed the build failure that I couldn't fix b/c I don't have permissions to his PR branch.

**Comment by jlevy44** [comment]
Oh I see. Thanks @cgreene

**Comment by jlevy44** [comment]
Do you know when will the new authors be added to the authors section of the manuscript? Thanks!

**Comment by cgreene** [comment]
Deep-review has once again made it to the top of my priority list. I'm excited to get this updated. I'm adding infrastructure for multi-version authors in #986. Thank you for your patience!

**Comment by delton137** [comment]
If you decide to add me, here is my info:   

- github: delton137
    name: Daniel C. Elton
    orcid: 0000-0003-0249-1387
    email: daniel.elton@nih.gov
    affiliations:
      - Clinical Center, National Institutes of Health, Bethesda, MD

**Comment by jlevy44** [comment]
We can send over our info as well. @AlexanderTitus @brockclarke 

Thanks for your work @cgreene !

**Comment by cgreene** [comment]
Can you each add yourselves by filing a pull request into `metadata.yaml`? It should work now that #986 has landed.

**Comment by delton137** [comment]
Hi,

I created a PR to add myself in metadata.yaml (edited it on github.com).

Thanks!!
-- Dan

--
Daniel C. Elton, Ph.D.
www.moreisdifferent.com
Email (work) : daniel.elton@nih.gov
Phone (cell)   : 518-409-3521

**Comment by cgreene** [comment]
- [ ] @dhimmel for #997 if there is a corresponding update to the methods section to describe how the manuscript is currently built
